### ISSUE TITLE
PHP 8.0 compatibility

### ISSUE THREAD

**Issue by ahlvd** [thephpleague/omnipay-sagepay#167]
We are looking into upgrading our project to PHP 8.0 but obviously this project officially does not support the latest version as of composer output:

```
- omnipay/sagepay is locked to version 3.2.3 and an update of this package was not requested.
- omnipay/sagepay 3.2.3 requires php ^5.6|^7 -> your php version (8.0.10) does not satisfy that requirement.
```

If this package is compatible to PHP 8.0 it would be nice if the composer requirements would be updated.

**Comment by sunscreem** [comment]
Wondering if there's any progress on this one folks?

**Comment by barryvdh** [comment]
Fixed with #169

**Comment by jamieburchell** [comment]
Please can you release?
